app/src/main/java/org/intelehealth/app/activities/notification/repository/NotificationRepository.kt

```diff
@@ -40,7 +40,8 @@ class NotificationRepository {
 
         nonDeletedNotificationList.forEach { notificationModel ->
             allVisitList.find { visitItem ->
-                visitItem.visitUuid == notificationModel.uuid
+                visitItem.visitUuid == if(notificationModel.notification_type == NotificationDbConstants.FOLLOW_UP_NOTIFICATION)
+                    notificationModel.uuid.split(" ")[0] else notificationModel.uuid
             }?.let { visitItem ->
                 notificationModel.apply {
```

Comment:
Model class variable name should be in camel case as per java naming conventions.
eg. first_name should be like firstName